pyracing/client.py

```diff
@@ -543,10 +589,18 @@ async def next_session_times(self, season_id):
         payload = {'season': season_id}
         url = ct.URL_SESSION_TIMES
         response = await self._build_request(url, payload)
-
-        return [upcoming_events.NextSessionTimes(x) for
-                x in response.json()["d"]["r"]]
-
+        
+        if not response.json():
+            return []
+        
+        mapping = response.json().get('m')
+        response_items = response.json().get('d').get('r')
+        
+        if mapping and response_items:
+            renamed_items = [self._rename_numerical_keys(ri, mapping) for ri in response_items]
+            
+            return [upcoming_events.NextSessionTimes(x) for x in renamed_items]
+    
```

Comment:
Same again, return value

Reply:
I won't comment any more, but there are a handful we need to make sure we have a return for

Reply:
See above